bazel/asn1c.bzl

```diff
@@ -158,7 +144,7 @@ def cc_asn1_library(
     """
     gen_name = name + "_genrule"
 
-    flags = "-pdu=all -fcompound-names -fno-include-deps -gen-PER"
+    flags = "-pdu=all -fcompound-names -fno-include-deps -gen-PER -no-gen-example"
```

Comment:
@nstng This was missing in the previous iteration.

Reply:
nice, building my test target is working now for me